packages/protocol/contracts/signal/SignalService.sol

```diff
@@ -194,6 +194,24 @@ contract SignalService is EssentialContract, ISignalService {
         }
     }
 
+    /// @inheritdoc ISignalService
+    function isSignalRootCached(
```

Comment:
We already have `getSyncedChainData`.

Reply:
"signal root", "state_root" are not something we expose through the signal service interface. They are values for the "kind" parameter.

Reply:
@dantaik do you think we shall close this then ?

Reply:
Removed redundant.

Reply:
you can keep this one `signalForChainData`